javascript/collapsible.js

```diff
@@ -1,90 +1,107 @@
 var tableRows = document.getElementsByTagName('tr');
 
-function toggleGroup(groupName) {
-  for (var i = 0; i < tableRows.length; i++) {
-    if (tableRows[i].title == groupName) {
-      if (tableRows[i].style.display == 'none') {
-        tableRows[i].style.display = 'table-row';
-      } else {
-        tableRows[i].style.display = 'none';
-      }
+function toggleGroup(groupName) 
+{
+
+    for (var i = 0; i < tableRows.length; i++) 
+    {
+        if (tableRows[i].title == groupName) 
+        {
+            if (tableRows[i].style.display == 'none') 
+            {
+                tableRows[i].style.display = 'table-row';
+            }
+            else 
+            {
+                tableRows[i].style.display = 'none';
+            }
+        }
     }
-  }
+
 }
 
-function collapseAllGroups() {
-  for (var i = 0; i < tableRows.length; i++) {
-    if (tableRows[i].title != '')
-      tableRows[i].style.display = 'none';
-  }
+function collapseAllGroups() 
+{
+
+    for (var i = 0; i < tableRows.length; i++) 
+    {
+        if (tableRows[i].title != '')
```

Comment:
You should avoid avoid statements like `if` without brackets as much as possible. It's too easily to wrongly change them:

	if (tableRows[i].title != '')
		tableRows[i].style.display = 'none';

	vs.

	if (tableRows[i].title != '')
		tableRows[i].style.display = 'none';
		tableRows[i].style.width = '40px';

	if (tableRows[i].title != '')
	{
		tableRows[i].style.display = 'none';
	}

	vs.

	if (tableRows[i].title != '')
	{
		tableRows[i].style.display = 'none';
		tableRows[i].style.width = '40px';
	}

Only the last two examples work always as intended.

Reply:
I have changed the if and else statement. I will keep this in mind if i come across more such thing in other section of the code. It is better to clean them then to keep it as is.